### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by Barasii**
### html-css-popup

[Demo](https://barasii.github.io/HTML-CSS-POPUP/) |
[Code base](https://github.com/Barasii/HTML-CSS-POPUP)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Barasii/HTML-CSS-Popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+    font-size: 1rem;
+    font-weight: 500;
+    line-height: 1.5;
+    color: #222;
+    background-color: #fff;
+}
+
+ul {
+    list-style: none;
+    list-style-type: none;
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/css/style.css]: `list-style` is a [shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). Don't need `list-style-type` + `list-style` here. Usually `list-style-type` will be enough.

File: submissions/Barasii/HTML-CSS-Popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+}
+
+.header {
+    height: auto;
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/css/style.css]: height will be `auto` by default

File: submissions/Barasii/HTML-CSS-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<! doctype html>
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/index.html]: space between `!` and `doctype` is not valid. This instruction will be recognized as bogus comment. See more info in [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fbarasii.github.io%2FHTML-CSS-POPUP%2F)

File: submissions/Barasii/HTML-CSS-Popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+    margin-top: 15px;
+    text-align: center;
+    width: 100%;
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/css/style.css]: width for blocks are 100% by default. Don't need it here

File: submissions/Barasii/HTML-CSS-Popup/css/style.css [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+}
+
+.navigation {
+    float: right;
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/css/style.css]: Do you really need `float` here?

**Comment by o-msh** [comment]
@madmaxWMFU @cchernysheva Thank you for review! Can you have anything else to add ?

**Comment by madmaxWMFU** [comment]
Everything is arranged!

File: submissions/Barasii/HTML-CSS-Popup/index.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+</head>
+<body>
+    <header class="header">
+        <nav class="navigation">
+            <ul class="navigation__menu">
+                <li class="navigation__item">
+                    <a href="#" alt="Email" class="navigation__link">Email</a>   
+                </li>
+                <li class="navigation__item">
+                    <a href="#" alt="Images" class="navigation__link">Images</a>
```
Comment (cchernysheva) [inline submissions/Barasii/HTML-CSS-Popup/index.html]: `a` tag doesn't have an `alt` attribute. More info in [spec](https://www.w3.org/TR/html50/text-level-semantics.html#the-a-element) (see content attributes list).

**Comment by o-msh** [comment]
@Barasii @madmaxWMFU @cchernysheva LGTM!

**Comment by Barasii** [comment]
@madmaxWMFU @o-msh @cchernysheva Many thanks for your reviews!!!

**Pull request merged**